crates/bevy_input/Cargo.toml

```
@@ -11,6 +11,7 @@ keywords = ["bevy"]
 [features]
 default = []
 serialize = ["serde"]
+serde = ["dep:serde", "dep:serde_derive"]
```

```suggestion
serialize = ["dep:serde", "dep:serde_derive"]
```
wouldn't it work to use the same feature instead of adding a new one?

---

I think the idea of a`serialize` feature was to not be explicit over `serde` dependency in case we want to replace it someday